### DISCUSSION TITLE
Anon chat logout

### DISCUSSION THREAD

**Discussion by VariableVixen** [night/betterttv#5987] [Ideas]
When anonymous chat mode is enabled, you can (and must in order to speak) use the `/join` custom command to login to chat and become visible. However, the only way to log out of chat is to reload the chat, either by closing and reopening the sidebar or reloading the stream page. Since it's clearly technically possible to logout of chat by enabling anonymous chat mode in the settings, which doesn't require a manual reload, it would be nice to have a command that could be used to do the same. If this already exists, it would be nice to have it mentioned under the toggle for anonymous chat mode in the settings, or (ideally) in the command autocomplete list when you start typing any command into twitch's chat input box.

**Comment by night** [comment]
You can use `/part` and `/join` to leave and rejoin the chat at any time temporarily.
